From florian.fainelli@telecomint.eu Tue Mar 25 11:31:45 2008
From: Florian Fainelli <florian.fainelli@telecomint.eu>
Date: Tue, 25 Mar 2008 17:32:16 +0100
Subject: USB: cp2101: Add identifiers for the Telegesys ETRX2USB
To: linux-usb@vger.kernel.org
Cc: gregkh@suse.de, craig@microtron.org.uk, Xavier Carcelle <xavier.carcelle@gmail.com>
Message-ID: <200803251732.18859.florian.fainelli@telecomint.eu>
Content-Disposition: inline


From: Florian Fainelli <florian.fainelli@telecomint.eu>

This patch adds support for the Telegesys ETRX2USB which
works fine with the cp2101 driver.

Signed-off-by: Florian Fainelli <florian.fainelli@telecomint.eu>
Tested-by: Xavier Carcelle <xavier.carcelle@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/serial/cp2101.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/usb/serial/cp2101.c
+++ b/drivers/usb/serial/cp2101.c
@@ -75,6 +75,7 @@ static struct usb_device_id id_table [] 
 	{ USB_DEVICE(0x10C4, 0x81E2) }, /* Lipowsky Industrie Elektronik GmbH, Baby-LIN */
 	{ USB_DEVICE(0x10C4, 0x81E7) }, /* Aerocomm Radio */
 	{ USB_DEVICE(0x10C4, 0x8218) }, /* Lipowsky Industrie Elektronik GmbH, HARP-1 */
+	{ USB_DEVICE(0x10c4, 0x8293) }, /* Telegesys ETRX2USB */
 	{ USB_DEVICE(0x10C4, 0xEA60) }, /* Silicon Labs factory default */
 	{ USB_DEVICE(0x10C4, 0xEA61) }, /* Silicon Labs factory default */
 	{ USB_DEVICE(0x10C4, 0xF001) }, /* Elan Digital Systems USBscope50 */
